charts/metrics-server/README.md

```diff
@@ -69,6 +69,17 @@ The following table lists the configurable parameters of the _Metrics Server_ ch
+| `addonResizer.enabled`               | If true, run the addon-resizer as a sidecar to automatically scale resource requests with cluster size.                                                                                                                                                          | `{}`                                       |
+| `addonResizer.image.repository`      | addon-resizer image repository                    | registry.k8s.io/autoscaling/addon-resizer              |
+| `addonResizer.image.tag`      | addon-resizer image tag                    | 1.8.14              |
+| `addonResizer.nanny.extraCPU`        | The amount of CPU to add per node.                                                                                                                                                                                                                               | 1m                                         |
+| `addonResizer.nanny.extraMemory`     | The amount of memory to add per node.                                                                                                                                                                                                                            | 2Mi                                        |
+| `addonResizer.nanny.memory`          | The base memory requirement.                                                                                                                                                                                                                                     | 15Mi                                       |
+| `addonResizer.nanny.minClusterSize`  | Specifies the smallest number of nodes resources will be scaled to.                                                                                                                                                                                              | 10                                         |
+| `addonResizer.nanny.pollPeriod`      | The time, in milliseconds, to poll the dependent container.                                                                                                                                                                                                      | 300000                                     |
+| `addonResizer.nanny.threshold`       | A number between 0-100. The dependent's resources are rewritten when they deviate from expected by more than threshold.                                                                                                                                          | 5                                          |
+| `addonResizer.resources`             | Resource requests and limits for the _nanny_ container.                                                                                                                                    | `{}`
```

Comment:
Could this ordering match the order in _values.yaml_?

Reply:
Of course :ok_hand: